dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -1511,4 +1511,17 @@ public WorkflowAction findActionRespectingPermissions(final String actionId,
 
 	}
 
+	@CloseDBIfOpened
+	public List<WorkflowAction> findInitialAvailableActionsByContentType(ContentType contentType, User user)
+			throws DotDataException, DotSecurityException{
+		final ImmutableList.Builder<WorkflowAction> actions = new ImmutableList.Builder<>();
+		final List<WorkflowScheme> schemes = findSchemesForContentType(contentType);
+		for(WorkflowScheme scheme: schemes){
+			List<WorkflowStep> steps = findSteps(scheme);
+			actions.addAll(findActions(steps.get(0), user));

```

Comment:
I think here you can do something like
steps.stream().findFirst().orElse(null);

If findActions receives null, return a Collections.emptyList();